### DISCUSSION TITLE
Model blending

### DISCUSSION THREAD

**Discussion by HubertII** [Comfy-Org/ComfyUI#542] [Q&A]
Hi, 

Could you tell me if it's possible to set weights on blending models? Let say I want to mix Gal Gadot look with Scarlett Johansson with proportion: 20% Gal, 80% Scarlett. In Automatic1111 I would use [Gal Gadot:Scarlett Johansson:0.2]. How to do it in Comfy?

**Comment by omar92** [comment]
you mean mix loras ??

**Reply by HubertII** [reply]
No, I mean setting proportion on the mix, let say I want a picture of a flower: 25% of tulip, 75% of rose. Is there a syntax that allow me to set that proportion? Something like [tulip|rose:0.25]?

**Comment by BlenderNeko** [comment]
the syntax you are describing is called prompt editing in A1111, which can be accomplished in Comfy by using 2 advanced K samplers where you feed the 2 samplers 2 different prompts. 
![image](https://user-images.githubusercontent.com/126974546/233651464-c8f923cf-5b91-423f-bfea-482f850d7635.png)
note the different start and end steps of the two samplers and the second sampler not adding noise

**Reply by BlenderNeko** [reply]
snap i should have also passed on the leftover noise to the second sampler
![ComfyUI_00262_](https://user-images.githubusercontent.com/126974546/233692086-9e37f93d-bd6c-464a-a109-aaeb1b265b90.png)
this image should hopefully contain a working example

**Reply by SadaleNet** [reply]
For those who doesn't want to create the KSampleAdvanced workflow manually, here's a solution that I just come up with: https://github.com/SadaleNet/CLIPTextEncodeA1111-ComfyUI

taabata has come up with another solution as well: https://github.com/taabata/Comfy_custom_nodes

Edit: Someone else told me about another solution by asagi4. I couldn't get this solution to work but it seems to be the most elegant one if I could get it work: https://github.com/asagi4/comfyui-prompt-control
